src/lane.cpp

```diff
@@ -0,0 +1,168 @@
+// Header
+#include "lane.hpp"
+
+// stlib
+#include <vector>
+#include <algorithm>
+
+bool Lane::init()
+{
+	std::vector<Vertex> vertices;
+	std::vector<uint16_t> indices;
+
+	FILE* mesh_file = fopen(mesh_path("salmon.mesh"), "r"); // TODO: Correct texture [YAO]

```

Comment:
Do we actually want to render the lane? The way I was thinking was that we would probably have a single static background image that included the intersection, but was just rendered like the blue background was for the salmon game. The lane would be more of a logical object that helps keep track of which cars are coming from which direction. I'm not committed to that idea, but I'm not sure how worthwhile it is to render each lane individually.

Reply:
Yeah, I agree with not rendering the lane. We can just give cars different curves for different maps and use the lane class to do what you mentioned.